.github/actions/README.md

```diff
@@ -0,0 +1,18 @@
+# Actions
+
+This directory contains code for GitHub Actions that are used in this repo's workflow.
+
+The actions are built with yarn like the rest of this repo.
+
+To build / update the actions, run this inside the action's directory:
+
+```
+yarn
+yarn run all
+```
+
+This will compile, lint and package the actions.
```

Comment:
[q] As it looks this only compiles and lints the action which is your CWD right? There are two the `get-changelog` and `check-pr` right?

Reply:
yes, I did not want to duplicate the readme so I've put it to the top level directory. will add a hint on the cwd.